### ISSUE TITLE
import-sql Error 124 when generating north-america

### ISSUE THREAD

**Issue by firstdorsal** [openmaptiles/openmaptiles#1126]
this get thrown when creating north-america

Everything completely up to date (repo,containers,pbf pulled 30min ago)

maybe related to #1125 but different error details

```log
psql:/sql/parallel/landcover.sql:419: NOTICE:  materialized view "ne_50m_antarctic_ice_shelves_polys_gen_z3" does not exist, skipping
psql:/sql/parallel/landcover.sql:430: NOTICE:  materialized view "ne_50m_antarctic_ice_shelves_polys_gen_z2" does not exist, skipping
psql:/sql/parallel/landcover.sql:442: NOTICE:  materialized view "ne_50m_glaciated_areas_gen_z4" does not exist, skipping
psql:/sql/parallel/landcover.sql:453: NOTICE:  materialized view "ne_50m_glaciated_areas_gen_z3" does not exist, skipping
psql:/sql/parallel/landcover.sql:464: NOTICE:  materialized view "ne_50m_glaciated_areas_gen_z2" does not exist, skipping
psql:/sql/parallel/landcover.sql:476: NOTICE:  materialized view "ne_10m_glaciated_areas_gen_z6" does not exist, skipping
psql:/sql/parallel/landcover.sql:487: NOTICE:  materialized view "ne_10m_glaciated_areas_gen_z5" does not exist, skipping
psql:/sql/parallel/landcover.sql:499: NOTICE:  materialized view "ne_10m_antarctic_ice_shelves_polys_gen_z6" does not exist, skipping
psql:/sql/parallel/landcover.sql:510: NOTICE:  materialized view "ne_10m_antarctic_ice_shelves_polys_gen_z5" does not exist, skipping
psql:/sql/parallel/landcover.sql:737: NOTICE:  Finished layer landcover
psql:/sql/parallel/building.sql:100: ERROR:  GEOSUnaryUnion: TopologyException: Input geom 1 is invalid: Self-intersection at or near point -8539739.7009410355 4727139.8920735931 at -8539739.7009410355 4727139.8920735931
CONTEXT:  PL/pgSQL function osm_building_block_gen1() line 14 at FOR over SELECT rows
xargs: sh: exited with status 255; aborting
124
Time: 1.428 ms
ALTER TABLE
Time: 2.360 ms
 update_osm_poi_point_agg 
--------------------------
 
(1 row)

Time: 109719.547 ms (01:49.720)
CREATE SCHEMA
Time: 0.880 ms
CREATE TABLE
...
Time: 1.261 ms
CREATE VIEW
Time: 1.228 ms
CREATE VIEW
Time: 1.481 ms
CREATE VIEW
Time: 1.255 ms
CREATE FUNCTION
Time: 2.363 ms
DO
Time: 0.376 ms
Time: 9442675.248 ms (02:37:22.675)
make: *** [Makefile:411: import-sql] Error 124

```

**Comment by TomPohys** [comment]
Hi, GEOS was updated in `latest` OpenMapTiles Tools. Please check if OMT is using correct (3.9.1) GEOS by `make psql` and `SELECT postgis_full_version();`

Thanks

**Comment by firstdorsal** [comment]
```sh
  postgis_full_version                                                                                       
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 POSTGIS="3.0.3 6660953" [EXTENSION] PGSQL="96" GEOS="3.7.1-CAPI-1.11.1 27a5e771" PROJ="Rel. 4.9.3, 15 August 2016" LIBXML="2.9.4" LIBJSON="0.12.1" LIBPROTOBUF="1.2.1" WAGYU="0.4.3 (Internal)"

```
it is version 3.7.1 indeed. i tried building the docker image as the one on dockerhub is not up to date but got the same version number from ```sh SELECT postgis_full_version();```. i think i'm not building it correct...

**Comment by firstdorsal** [comment]
i built the docker image from the repo and GEOS is now using 3.9.1:
```sh
 postgis_full_version                                                                                  
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 POSTGIS="3.1.2 cbe925d" [EXTENSION] PGSQL="96" GEOS="3.9.1-CAPI-1.14.2" PROJ="Rel. 4.9.3, 15 August 2016" LIBXML="2.9.4" LIBJSON="0.12.1" LIBPROTOBUF="1.2.1" WAGYU="0.5.0 (Internal)"
```
the problem is still the same for antarctica:
```log
[Jun 15 22:52:18] [INFO] [PostGIS] Rotating osm_border_disp_linestring_gen_z1 from import -> public -> backup
[Jun 15 22:52:18] [INFO] [PostGIS] Rotating tables took: 338.485613ms
[Jun 15 22:52:18] [INFO] Imposm took: 24.795979975s
 
-------------------------------------------------------------------------------------
====> : Start importing border antarctica data into PostgreSQL using osmborder
      : Source code: https://github.com/pnorman/osmborder
      : Data license: http://www.openstreetmap.org/copyright
      : Thank you Paul Norman
Starting postgres docker compose target using default image (no recreate if exists)
# If CSV borders file already exists, use it without re-parsing
docker-compose run --rm --user=0:0 openmaptiles-tools sh -c \
        'pgwait && import-borders $([ -f "data/borders/lines.csv" ] && echo load data/borders/lines.csv || echo import data/antarctica.osm.pbf)'
Creating openmaptiles_openmaptiles-tools_run ... 
Creating openmaptiles_openmaptiles-tools_run ... done
Skipping broken references cleanup because BORDERS_CLEANUP is not set to 'true'
Enable borders cleanup if you see 'location for one or more nodes not found in node location index' error
Filtering data/antarctica.osm.pbf into data/borders/filtered.pbf
/usr/src/app/import-borders: line 91:    14 Segmentation fault      (core dumped) osmborder_filter -o "$BORDERS_PBF_FILE" "$IMPORT_PBF_FILE"
139
make: *** [Makefile:406: import-borders] Error 139
```
BUT north america  works now: (at least this stage)
```log
root@server1:~/openmaptiles# make import-borders
Starting postgres docker compose target using default image (no recreate if exists)
Creating openmaptiles_postgres_1 ... done
Detected area=north-america based on finding a 'data/north-america.osm.pbf' file. Use 'area' parameter or environment variable to override.
# If CSV borders file already exists, use it without re-parsing
docker-compose run --rm --user=0:0 openmaptiles-tools sh -c \
        'pgwait && import-borders $([ -f "data/borders/lines.csv" ] && echo load data/borders/lines.csv || echo import data/north-america.osm.pbf)'
Creating openmaptiles_openmaptiles-tools_run ... done
Skipping broken references cleanup because BORDERS_CLEANUP is not set to 'true'
Enable borders cleanup if you see 'location for one or more nodes not found in node location index' error
Filtering data/north-america.osm.pbf into data/borders/filtered.pbf
Creating a CSV borders file data/borders/lines.csv
There were 0 warnings.
There were 0 errors.
Importing data/borders/lines.csv into postgres:5432/openmaptiles as table osm_border_linestring...
NOTICE:  table "osm_border_linestring" does not exist, skipping
DROP TABLE
CREATE TABLE
CREATE INDEX
COPY 217291
```



thank you

**Comment by TomPohys** [comment]
Please, try set up `BORDERS_CLEANUP=true` in [`.env` file](https://github.com/openmaptiles/openmaptiles/blob/master/.env#L33)

**Comment by firstdorsal** [comment]
fully recreated the server each time and tried ```BORDERS_CLEANUP=true``` with and without building the newest postgis image with GEOS="3.9.1-CAPI-1.14.2"
same error
```log
-------------------------------------------------------------------------------------
====> : Start importing border antarctica data into PostgreSQL using osmborder
      : Source code: https://github.com/pnorman/osmborder
      : Data license: http://www.openstreetmap.org/copyright
      : Thank you Paul Norman
Starting postgres docker compose target using default image (no recreate if exists)
# If CSV borders file already exists, use it without re-parsing
docker-compose run --rm --user=0:0 openmaptiles-tools sh -c \
	'pgwait && import-borders $([ -f "data/borders/lines.csv" ] && echo load data/borders/lines.csv || echo import data/antarctica.osm.pbf)'
Creating openmaptiles_openmaptiles-tools_run ... 
Creating openmaptiles_openmaptiles-tools_run ... done
Cleaning-up broken references from data/antarctica.osm.pbf into data/borders/cleanup.pbf
Set BORDERS_CLEANUP=false to skip the cleanup -- it is not needed for the full planet and Geofabrik downloads
Filtering data/borders/cleanup.pbf into data/borders/filtered.pbf
/usr/src/app/import-borders: line 91:    18 Segmentation fault      (core dumped) osmborder_filter -o "$BORDERS_PBF_FILE" "$IMPORT_PBF_FILE"
139
make: *** [Makefile:406: import-borders] Error 139
```

thank you

**Comment by TomPohys** [comment]
This is very specific area, the Antartica have a problem with boundary. Based on [taginfo](https://taginfo.openstreetmap.org/tags/boundary=administrative#map) there is none. The `osmborder` is looking for tag `boundary=administrative`

**Comment by TomPohys** [comment]
Antarctica is now able to import without `import-border` make target. 

Fixed by #1213
